source/whats-new.txt

```diff
@@ -20,6 +21,47 @@ Learn what's new in:
 * :ref:`Version 4.1 <version-4.1>`
 * :ref:`Version 4.0 <version-4.0>`
 
+.. _version-4.7:
+
+What's New in 4.7
+-----------------
+
+New features of the 4.7 driver release include:
+
+- Improved change stream support
+
+  - Enhanced pre-image and post-image support for :ref:`change streams <java-usage-watch>`.
+    Change stream watch helpers now accept ``whenAvailable`` and ``required``
+    for the fullDocument option.
+  - Added a new ``fullDocumentBeforeChange`` option, which accepts
+    ``whenAvailable`` and ``required``.
+  - Change events may now include a ``fullDocumentBeforeChange`` response field.
```

Comment:
Suggestion: clarify what it means to "may now include" as it doesn't seem definitive.